pkg/state/router.go

```diff
@@ -620,7 +620,7 @@ func entityToMap(v interface{}) (map[string]interface{}, error) {
 	converted := make(map[string]interface{}, len(data))
 
 	for k, v := range data {
-		switch vt := v.(type) {
+		switch v.(type) {
```

Comment:
If you retained the vt variable, you could avoid the cast.   That's really why type switches exist.